collections/_posts/2025-01-14-TinyUNet.md

```diff
@@ -0,0 +1,146 @@
+---
+layout: review
+title: "TinyU-Net: Lighter yet Better U-Net with Cascaded Multi-Receptive Fields"
+tags: medical-image-segmentation lightweight-neural-networks multi-receptive-fields health-equity
+author: "Maylis Jouvencel"
+cite:
+    authors: "Junren Chen, Rui Chen, Wei Wang, Junlong Cheng, Lei Zhang, Liangyin Chen"
+    title:   "TinyU-Net: Lighter yet Better U-Net with Cascaded Multi-Receptive Fields"
+    venue:   "MICCAI, 2024"
+pdf: "https://papers.miccai.org/miccai-2024/paper/2191_paper.pdf"
+---
+
+# Notes
+* Link to the code [here](https://github.com/ChenJunren-Lab/TinyU-Net)
+
+# Highlights
+* The goal of the paper is to propose a lightweight segmentation network without compromising on the quality of the segmentation maps. 
+* The authors propose TinyU-Net, a network with a UNet-like architecture based on a new CMRF module (Cascade Multi-Receptive Fields).
+
+
+
+# Introduction
+
+Developping models which can run on **limited resources** is one of the challenges in the medical field to guarantee **health equity**.
+Those **lightweight** models usually try to work on a reduced number of parameters and computation complexity. For the segmentation task however, 
+the existing methods produce results with lower quality than their non-lightweight counterparts due to the a lower representation capacity.
+
+There is therefore a need to develop lightweight models which also improve the segmentation quality.
+
+
+# Method
+
+![](/collections/images/TinyUNet/architecture.jpg)\
+*Figure 1: Details of the CMRF (left part of the figure) and the architecture of the TinyUNet (right part of the figure).*
+
+## CMRF
+
+**Main idea:** fuse information from multi-receptive fields with a lightweight cascading strategy
+
+> **Reminders:** 
+- A **depthwise convolution** applies a single convolutional filter for each input channel. Each channel is kept separate contrary to regular convolutions which mix the different channels. This leads to fewer parameters and number of operations. \
+<img src="/collections/images/UNeXt/depthwise-conv.jpg" alt="DWconv" width="60%"/> \
+*Figure 2: Depth-wise convolution, from [^1]*
+- A **pointwise convolution** is a convolution that uses a 1x1 kernel.
```

Comment:
Maybe clarify that a pointwise convolution is not a depthwise convolution but a standard convolution.